make.bat

```diff
@@ -26,19 +26,21 @@ choice /c:1234567QpbD>NUL
 :: Debug = OFF [default]
 
 :: OPTIONS
-if errorlevel 11 goto debug
```

Comment:
This was organized in descending numerical order for a reason. Several parts of the batch file break (i.e. the quit option, file moving, etc) when you order these in an ascending fashion due to the order in which the labels are parsed.

Reply:
@Counterply, putting %errorlevel% instead of errorlevel may solve the problem.

Reply:
I think you misunderstand me. There is no problem; it works as intended. No change is necessary.

Reply:
Ok. I thought it would be better for someone who wants to explore the program. (Note that with not `%errorlevel%`, this didn't work.)